dipy/fixes/argparse.py

```diff
@@ -1909,6 +1909,10 @@ def consume_positionals(start_index):
         # if we didn't use all the Positional objects, there were too few
         # arg strings supplied.
         if positionals:
+            #  printing user friendly help message to tell about missing
+            #  arguments.
+            print("Too few arguments. Program", self.prog, "expects arguments"
```

Comment:
Can you some "\n\n"

Reply:
@skoudoro: Thanks for pointing this out. I have added the line gaps and committed. Thanks.